sim/priest/mindflay.go

```diff
@@ -0,0 +1,66 @@
+package priest
+
+import (
+	"time"
+
+	"github.com/wowsims/tbc/sim/core"
+	"github.com/wowsims/tbc/sim/core/stats"
+)
+
+const SpellIDMF int32 = 25387
+
+const TagMF2 = 2
+const TagMF3 = 3
+
+func (priest *Priest) newMindflayTemplate(sim *core.Simulation) core.SimpleSpellTemplate {
+	baseCast := core.Cast{
+		Name:           "Mind Flay",
+		CritMultiplier: 1.5,
+		SpellSchool:    stats.ShadowSpellPower,
+		Character:      &priest.Character,
+		BaseManaCost:   230,
+		ManaCost:       230,
+		CastTime:       0,
```

Comment:
AFAICT this will just set a basic GCD of 1.5s, but isnt the channel time longer than that?

Reply:
We "channel" by using a dot + wait.  The GCD is still invoked when starting a channel. The actual cast time is 0.

Reply:
Ah I see, need to refactor this later to have the wait be part of the spell.